makefiles/color.inc.mk

```diff
@@ -5,7 +5,7 @@ COLOR_GREEN  :=
 COLOR_RED    :=
 COLOR_PURPLE :=
 COLOR_RESET  :=
-COLOR_ECHO   := /bin/echo
+COLOR_ECHO   := echo
```

Comment:
Wouldn't it rather be better to check for that? There must be a reason, why a full path is chosen here.

Reply:
Maybe the reasoning was that some built-in version might not support the `-e` option (POSIX [`echo`](https://pubs.opengroup.org/onlinepubs/009695399/utilities/echo.html) does not have any options), but I don't see why in that case it would be save to assume the `echo` binary does.

Reply:
> […] but I don't see why in that case it would be save to assume the echo binary does.

Given the current situation, the question would rather be why it isn't ;-P

Reply:
We could just always set `SHELL=bash` when using `COLOR_ECHO   := echo -e` like we do in the `ifeq ($(OS),Darwin)` case.

Reply:
>We could just always set SHELL=bash when using COLOR_ECHO := echo -e like we do in the ifeq ($(OS),Darwin) case.

hmm, I would prefer a solution that does not add *more* constraints to the shell in use (: